vdr/vdr.go

```diff
@@ -172,7 +172,7 @@ func (r VDR) Update(id did.DID, current hash.SHA256Hash, next did.Document, _ *t
 		return types.ErrDeactivated
 	}
 
-	if err = CreateDocumentValidator().Validate(next); err != nil {
+	if err = NetworkDocumentValidator().Validate(next); err != nil {
```

Comment:
shouldn't this be the `ManagedDocumentValidator`?

Reply:
This PR introduses the validator, the next PR applies it + removes checks in Didman and elsewhere that are now redundant. It is split to minimise conflicts with VDR redesign.

Reply:
I wanted to split approval of the concept and implementing it, but I've added it where appropriate.

Reply:
changed the design a bit in [2faee38](https://github.com/nuts-foundation/nuts-node/pull/1769/commits/2faee38d15038389d35bed2baaa47ff9f536a5a0) to deal with the different formats of ServiceEndpoints